roles/devscripts/tasks/main.yml

```diff
@@ -49,6 +49,8 @@
   poll: 0
 
 - name: Check async deploy status
+  when:
+    - not cifmw_devscripts_dry_run | bool
```

Comment:
we could also `when: _devscripts_deploy.ansible_job_id is defined` - but this one should be good as well.